scripts/create-symlinks.sh

```diff
@@ -142,6 +142,7 @@ sudo ln -sf $SOURCEDIR/Config/keys/dlms/osgp-protocol-adapter-rsa-public.key $TA
 sudo ln -sf $SOURCEDIR/Config/keys/dlms/osgp-secret-management-jre.key $TARGETDIR/smartmetering/keys/platform/
 mkdir -p $TARGETDIR/smartmetering/keys/application
 sudo ln -sf $SOURCEDIR/Config/keys/application/smartmetering-rsa-public.key $TARGETDIR/smartmetering/keys/application/smartmetering-rsa-public.key
+sudo ln -sf $SOURCEDIR/Config/keys/application/smartmetering-rsa-private.key $TARGETDIR/smartmetering/keys/application/smartmetering-rsa-private.key
```

Comment:
This places a private key that is not intended for regular use by GXF in the filesystem with keys that are intended for regular use by the platform.
Wouldn't it be clearer to keep the key that is not for GXF further away, perhaps like:
```
mkdir -p $TARGETDIR/test/smartmetering/keys/application
sudo ln -sf $SOURCEDIR/Config/keys/application/smartmetering-rsa-private.key $TARGETDIR/test/smartmetering/keys/application/smartmetering-rsa-private.key
```

Reply:
I'm not sure about this change, this will complicate things a lot.
The last months we spent a lot of time to get our development aligned with PRD. This means that I would prefer the paths to be the same, but the contents to be different. If we need a test key, then I would prefer generating a test keypair and linking the test source to the actual folder.

`sudo ln -sf $SOURCEDIR/Config/keys/test/application/smartmetering-rsa-private.key $TARGETDIR/smartmetering/keys/application/smartmetering-rsa-private.key
`

Reply:
I am confused about what you are saying about aligning with PRD, and preferring to generate a test key pair (in Config/test instead of Config/keys).
What can be found under Config/keys are keys for development and testing purposes.
In production you may never even see the smartmetering-rsa-private.key anywhere near the GXF deployment.
It is the private key of an application used by some organization on top of GXF. This may even be a different organization than the one managing the platform.

A fresh install of the platform would not even have knowledge of the smartmetering-rsa-public.key. This will be configured when an organization is added with an application that will retrieve device keys. Other such public keys may be added when more organizations run their separate smart metering applications on the same instance of GXF.